analyzer/tools/build-logger/src/ldlogger-tool-gcc.c

```diff
@@ -16,23 +16,10 @@
 #include <ctype.h>
 #include <unistd.h>
 #include <string.h>
-/**
- * States for GCCargument parser.
- */
-typedef enum _GccArgsState
-{
-  /**
-   * Normal state (default).
-   */
-  Normal,
-  /**
-   * After a -o paramater.
-   */
-  InOutputArg
-} GccArgsState;
 
 /**
- * List of file extensions accepted as source file.
+ * List of file extensions accepted as source file. Binaries can also be
```

Comment:
There are two more file extensions which can be source files (https://linux.die.net/man/1/g++):
- file.cc
- **file.cp**
- file.cxx
- file.cpp
- file.CPP
- **file.c++**
- file.C